src/expression-language/index.ts

```diff
@@ -423,16 +423,20 @@ export const ruleExpressionLanguage = function(
     FILTER_VALUES: filterValues,
     INCLUDES_VALUES: filterValues,
     GET: (arg1, arg2) => {
-      const key = string(arg1());
+      // this needs to accept a string or number
+      const key = arg1();
```

Comment:
Can we make sure it either gets run through `number()` or `string()`? Consistency, etc.

Reply:
So, determine the type then run it through one of those?